web/modules/custom/ai_screening_project_track/src/Entity/ProjectTrack.php

```diff
@@ -174,8 +179,30 @@ public function getDescription(): string {
   /**
    * {@inheritdoc}
    */
-  public function getProjectTrackEvaluation(): string {
-    return $this->get('project_track_evaluation')->getString();
+  public function setProjectTrackEvaluation(Evaluation $evaluation): self {
+    $this->set('project_track_evaluation', $evaluation->value);
+
+    return $this;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getProjectTrackEvaluation($ignoreOverridden = FALSE): string {
+    if ($ignoreOverridden) {
+      return $this->get('project_track_evaluation')->getString();
+    }
+    else {
+      return !empty($this->getProjectTrackEvaluationOverridden()) ? $this->getProjectTrackEvaluationOverridden() : $this->get('project_track_evaluation')->getString();
```

Comment:
Could be rewritten to 

```suggestion
      return $this->getProjectTrackEvaluationOverridden() ?: $this->getProjectTrackEvaluation(TRUE);
```